routers/private/repository.go

```diff
@@ -56,15 +56,15 @@ func GetRepository(ctx *macaron.Context) {
 func GetActivePullRequest(ctx *macaron.Context) {
 	baseRepoID := ctx.QueryInt64("baseRepoID")
 	headRepoID := ctx.QueryInt64("headRepoID")
-	baseBranch, err := url.QueryUnescape(ctx.QueryTrim("baseBranch"))
+	baseBranch, err := url.PathUnescape(ctx.QueryTrim("baseBranch"))
```

Comment:
`ctx.QueryTrim` will already unescape its contents so you don't actually need to unescape these.

Reply:
removed that call. Included a check if baseBranch is empty to preserve the error handling at this point.